cluster-provision/k8s/1.18/provision.sh

```diff
@@ -205,47 +205,6 @@ spec:
           type: spc_t
 EOF
 
-
-pod_cidr="10.244.0.0/16"
```

Comment:
Please split this first commit in two. First move the code down, then replace the values you need to. It is hard to tell what actually changed here.

Reply:
omg HD

Reply:
done

Reply:
same :/

Reply:
see please the 2nd commit, just there i moved it, already done, please refresh and see the 2 commits